lib/utils.rb

```diff
@@ -25,7 +25,7 @@ def self.normalize_uri(uri)
     begin
       URI(uri)
     rescue URI::Error
-      URI(uri.to_s.gsub(/[^\-_.!~*'()a-zA-Z\d;\/?:@&=+$,\[\]]+/) { |unsafe|
+      URI(uri.to_s.gsub(/[^\-_.!~*'()a-zA-Z\d;\/?:@&=+$,]+/) { |unsafe|
```

Comment:
Thanks. Unfortunately, I can't really test this fix as I use Huginn with Docker. However any solution is acceptable to me as long as it fixes the issue with square brackets.